portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,11 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.project-container {
+    margin-bottom: 15px;
+}
+
+hr.solid {
+    border-top: 2px solid #000;
+  }
```

Comment:
Use a trailing newline to help make `git diff` happier. Also why the additional leading blank?